com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/BundleHelper.java

```diff
@@ -1131,7 +1131,12 @@ public static File buildEngineRemote(ExtenderClient extender, String platform, S
                     buildError = FileUtils.readFileToString(logFile);
                     parseLog(platform, buildError, issues);
                     MultipleCompileException exception = new MultipleCompileException("Build error", e);
+                    IResource projectResource = project.getGameProjectResource();
                     IResource extManifestResource = ExtenderUtil.getResource(allSource.get(0).getPath(), allSource);
+                    if (extManifestResource == null) {
+                        extManifestResource = projectResource;
+                    }
```

Comment:
The actual issue: If the fallback resource was null, then we got an exception further down, messing up the error reporting.